test/spec/ol/style/style.test.js

```diff
@@ -64,6 +68,9 @@ describe('ol.style.Style', function () {
       expect(original.getImage().getRadius()).to.eql(
         clone.getImage().getRadius()
       );
+      expect(original.getRenderer().toString()).to.eql(
+        clone.getRenderer().toString()
+      );
```

Comment:
I would assume they are not only identical by string association but also by reference. So in my opinion
```
expect(original.getRenderer()).to.eql(clone.getRenderer());
```
would be correct.